firmware/tunerstudio/rusefi.input

```diff
@@ -2191,7 +2198,9 @@ cmd_set_engine_type_default					= "w\x00\x31\x00\x00"
 ;			Board->Connection
 	dialog = tsPort, "TunerStudio Port"
 		field = "tunerStudioSerialSpeed",				tunerStudioSerialSpeed
-		field = "Use PC10/PC11 serial?",				useSerialPort
+		field = "Use UART/TTL serial?",					useSerialPort
```

Comment:
unrelated change?

Reply:
I pulled this file again to remove some bug fixes, I tought I pasted only the boost control.
Are this a diff now?

Reply:
It had been edited in the meantime, from the one I originally pushed.

I dont know git good enough to know if this is a current difference..